### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by camagallen**
## Overview
The focus of this PR is to flesh out docstrings on public-facing functions and to add clarity to some components of the code. Additionally, the "feature_table" function was moved out of the kdd_2020_tutorial_helpers.py module into utils.py so that it can be used publicly.

Resolves: [AB#13807](https://dev.azure.com/kensci/91234890-e1c9-4679-971f-09afd8516db1/_workitems/edit/13807)

## How was this update tested?
pytest

## Checklist
- [X] I have read the [Contributing](https://github.com/KenSciResearch/docs/code_contributions/CONTRIBUTING.md) guidelines
- [X] I performed a self-review of my own code
- [X] The CHANGELOG and version.json have been updated
- [X] Documentation has been updated to reflect the changes
    - [X] The code has clear and helpful comments, particularly in hard-to-understand areas
    - [X] READMEs, Tutorials/Examples, and links have been updated as appropriate
- [X] Tests have been added to prove that the fix is effective or that the feature works
- [X] Pytest was run; new and existing unit tests pass locally with my changes
- [X] I have made note of any breakages or dependencies (if applicable)
- [X] Any dependent changes have been merged and published in downstream modules
- [X] These changes generate no new warnings

### CODE REVIEW

File: docs/publications/KDD2020-FairnessInHealthcareML-TutorialNotebook.ipynb [hunk 1]
```diff
@@ -171,7 +171,7 @@
    "outputs": [],
    "source": [
     "# Helpers from local folder\n",
-    "from fairmlhealth import reports, tutorial_helpers as helpers\n",
+    "from fairmlhealth import reports, __kdd2020_tutorial_helpers as helpers\n",
```
Comment (kenviku) [inline docs/publications/KDD2020-FairnessInHealthcareML-TutorialNotebook.ipynb:174]: is there a reason you want to keep the reference for kdd2020 in tutorial? Is this for backward compatibility?

File: fairmlhealth/__kdd2020_tutorial_helpers.py [hunk 2]
```diff
@@ -12,14 +12,16 @@
...
-
+from warnings import warn
 
 '''
 Global variable for backward compatibility with KDD2020 tutorial. Used to
     reduce verbosity of comparison tables.
 '''
 TUTORIAL_ON = False
-
+DeprecationWarning("The KDD2020 tutorial notebook is no longer supported." +
```
Comment (kenviku) [inline fairmlhealth/__kdd2020_tutorial_helpers.py:22]: 👍

File: fairmlhealth/reports.py [hunk 3]
```diff
@@ -1013,7 +1040,7 @@ def set_measure_labels(self, df):
             label_type = "columns"
         if label_type == "columns":
             labels = df.columns.tolist()
-        return df, labels, label_type
+        return labels, label_type
```
Comment (kenviku) [inline fairmlhealth/reports.py:1043]: `df` not required? Its hard to see here what it was used for.

Comment (camagallen) [reply fairmlhealth/reports.py:1043]: Previously it went through some formatting, and for a short time after it was still returned as a temporary validation, but these needs are addressed elsewhere.

File: docs/publications/KDD2020-FairnessInHealthcareML-TutorialNotebook.ipynb (hunk 1 above)
Comment (camagallen) [reply docs/publications/KDD2020-FairnessInHealthcareML-TutorialNotebook.ipynb:174]: correct

**Review by kenviku — APPROVED** [review]
LGTM

**Pull request merged**